src/config-hardware/audio-apps.md

```diff
@@ -0,0 +1,39 @@
+# Audio Applications & Extensions
+
+These applications provide control and customization settings for audio devices beyond the `Settings` menu.
+
+---
+
+## EasyEffects
+
+[EasyEffects](https://github.com/wwmm/easyeffects) is a third-party application that can apply audio effects and filters to audio input and output streams. You can download and install EasyEffects from the Pop!\_Shop.
+
+![EasyEffects](/images/audio-mic/easyeffects.png)
+
+## Audio Output Switcher
+
+[Audio Output Switcher](https://extensions.gnome.org/extension/751/audio-output-switcher/) is a GNOME extension that adds selectable audio input/output devices to the System menu. See [this section](/customize-pop/gnome-tweaks-extensions/gnome-extensions.md) for more information about installing GNOME Extensions.
+
+![Audio Output Switcher](/images/audio-mic/output-switcher.png)
```

Comment:
I think this is the first time I've seen menu items blurred out in a screenshot. Why's that? (If it's supposed to be removing your personal WiFi network name to make the screenshot more generic, you've still got Slack and Mattermost open anyway.)

Reply:
This blur's out the coffee shop WiFi I was on and also the VPN server location I was using. For most screenshots I use my VM with the "Pop User" username. I try to keep info in screenshots as neutral and non-identifying as possible, but in this case I needed to use the host to show multiple sound output sources. I don't really see the Slack and Mattermost icons as having the same identifying info as those other items. It's not a hill I'll die on so if you think the blur is distracting I can retake the screenshot.

Reply:
I would recommend retaking it with no blur and without the non-default app icons, for consistency's sake.

Reply:
Done